docs/release-process/CHECKLIST.md

```diff
@@ -0,0 +1,84 @@
+## Check list
```

Comment:
This PR should also create a sample lokocfg file with all the cluster features and component features. All parameterised using hcl variables and a sample lokocfg.vars file which has empty vars. So the user (fellow developer) has to just edit those values and get going.

Reply:
Now we have different platforms, we can have lokocfg files for two platforms viz. packet and aws. For that we might have to put the files in two different sub dirs.